src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -360,38 +360,6 @@ TEST(OVClassBasicTest, smoke_SetConfigAutoNoThrows) {
     EXPECT_EQ(value, ov::hint::Priority::HIGH);
 }
 
-TEST(OVClassBasicTest, smoke_SetConfigWithNoChangeToHWPluginThroughMetaPluginNoThrows) {
-    ov::Core ie = createCoreWithTemplate();
-    int32_t preValue = -1, curValue = -1;
-
-    ASSERT_NO_THROW(ie.set_property(CommonTestUtils::DEVICE_CPU, {ov::num_streams(20)}));
-    ASSERT_NO_THROW(curValue = ie.get_property(CommonTestUtils::DEVICE_CPU, ov::num_streams));
-    EXPECT_EQ(curValue, 20);
-    std::vector<std::string> metaDevices = {CommonTestUtils::DEVICE_AUTO,
-                                            CommonTestUtils::DEVICE_MULTI,
-                                            CommonTestUtils::DEVICE_HETERO};
-
-    for (auto&& metaDevice : metaDevices) {
-        ASSERT_NO_THROW(preValue = ie.get_property(CommonTestUtils::DEVICE_CPU, ov::num_streams));
-        ASSERT_NO_THROW(
-            ie.set_property(metaDevice, {ov::device::properties(CommonTestUtils::DEVICE_CPU, ov::num_streams(20))}));
-        ASSERT_NO_THROW(curValue = ie.get_property(CommonTestUtils::DEVICE_CPU, ov::num_streams));
-        EXPECT_EQ(curValue, preValue);
-    }
-    ASSERT_THROW(ie.set_property(CommonTestUtils::DEVICE_CPU,
-                                 {ov::device::properties(CommonTestUtils::DEVICE_CPU, ov::num_streams(20))}),
-                 ov::Exception);
-    ASSERT_THROW(ie.set_property(CommonTestUtils::DEVICE_GPU,
-                                 {ov::device::properties(CommonTestUtils::DEVICE_CPU, ov::num_streams(20))}),
-                 ov::Exception);
-    ASSERT_THROW(ie.set_property(CommonTestUtils::DEVICE_GPU,
-                                 {ov::device::properties("GPU.0", ov::num_streams(20))}),
-                 ov::Exception);
-    ASSERT_THROW(ie.set_property("GPU.0",
-                                 {ov::device::properties("GPU.0", ov::num_streams(20))}),
```

Comment:
do we have a test for new behavior?

Reply:
You are right. Will add corresponding test case soon.